libs/model/src/thread/CreateThread.command.ts

```diff
@@ -90,7 +90,9 @@ export function CreateThread(): Command<
       isAuthorized({ action: schemas.PermissionEnum.CREATE_THREAD }),
       verifyThreadSignature,
     ],
-    body: async ({ actor, payload }) => {
+    body: async ({ actor, payload, auth }) => {
+      const { address } = mustBeAuthorized(actor, auth);
```

Comment:
my first time seeing the auth context passed here– curious to learn more about that.